src/main/java/appeng/block/misc/BlockInterface.java

```diff
@@ -52,6 +57,26 @@ public BlockInterface()
 	}
 
 	@Override
+	protected IProperty[] getAEStates()
+	{
+		return new IProperty[] { AE_BLOCK_FORWARD, AE_BLOCK_UP, OMNIDIRECTIONAL };
```

Comment:
Good question. That might also be something related to synchronizing and the package structure in general. The interface could easily be a subclass of `AEBaseNetworkTileEntity`, which handles commonly shared states like `isActive`, `isPowered` and so on. You do not want to touch every single subclass, just because `AEBaseNetworkTileEntity` got a new property for some reasons.